honey-token-tools/disseminate_honeytokens.py

```diff
@@ -0,0 +1,224 @@
+import argparse
+import json
+import os
+
+from common import (
+    PullRequestInfo,
+    get_branch_name_from_commit_message,
+    get_saas_url_for_vcs,
+    GITGUARDIAN_SAAS,
+    RepositoryInfo,
+    HoneyTokenCreationError,
+    PullRequestCreationError,
+    CredentialsValidationError,
+    RepositoryAccessError,
+)
+from utils import get_client_for_vcs
+from gg_client import GGClient
+
+
+def parse_vcs_url(vcs_url: str | None, vcs: str) -> str | None:
+    if not vcs_url:
+        vcs_url = os.getenv("VCS_URL")
+        if not vcs_url:
+            vcs_url = get_saas_url_for_vcs(vcs)
+    return vcs_url.strip("/")
+
+
+def parse_vcs_token(vcs_token: str | None) -> str | None:
+    return os.getenv("VCS_TOKEN") if not vcs_token else vcs_token
+
+
+def parse_gitguardian_url(gitguardian_url: str | None) -> str | None:
+    if not gitguardian_url:
+        gitguardian_url = os.getenv("GITGUARDIAN_URL")
+        if not gitguardian_url:
+            gitguardian_url = GITGUARDIAN_SAAS
+    return gitguardian_url.strip("/")
+
+
+def parse_gitguardian_token(gitguardian_token: str | None) -> str | None:
+    return (
+        os.getenv("GITGUARDIAN_TOKEN") if not gitguardian_token else gitguardian_token
+    )
+
+
+def main():
+    parser = argparse.ArgumentParser(
+        description="Script to disseminate Honey Tokens in your repositories via pull requests."
+    )
+
+    parser.add_argument(
+        "--vcs",
+        choices=["github", "gitlab", "bitbucket", "ado"],
```

Comment:
should gitlab/bitbucket/ado be removed for now until it's implemented?
To avoid them appearing in the help of the script.